src/js/components/Paragraph/stories/Themed.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Paragraph } from 'grommet';
+import { deepMerge } from 'grommet/utils';
+import { grommet } from 'grommet/themes';
+import isChromatic from 'chromatic/isChromatic';
+
+const customTheme = deepMerge(grommet, {
+  paragraph: {
+    font: {
+      family: 'Comic Sans MS',
+    },
+  },
+});
+
+const All = () => (
+  <Grommet theme={customTheme}>
+    <Paragraph>
+      The font family for this paragraph is being defined by a custom theme.
+    </Paragraph>
+  </Grommet>
+);
+
+if (isChromatic && document.fonts) {
```

Comment:
What is `document.fonts` and why it is needed?

Reply:
I added `document.fonts` as a workaround suggested[ here ](https://www.chromatic.com/docs/resource-loading#loading-custom-fonts) to try and get the chromatic snapshots to show the font but the snapshot still is not capturing the font